lively.ide/world-commands.js

```diff
@@ -11,9 +11,24 @@ import { LoadingIndicator } from "lively.components";
 import { createMorphSnapshot } from "lively.morphic/serialization.js";
 import { interactivelyFreezeWorld } from "lively.freezer";
 import { resource } from "lively.resources";
+import { BrowserModuleTranslationCache } from "lively.modules/src/instrumentation.js";
 
 var commands = [
 
+  {
+    name: "clear storage and reload",
+    exec: async function() {
+      var proceed = await $world.confirm("Proceeding will clear all local storage and reload the page! Make sure if you want to save your world before proceeding.");
```

Comment:
I would suggest this formatting of the prompt: 
```
$world.confirm(["Caution\n", {}, "Proceeding will clear all local storage and reload the page! Make sure if you want to save your world before you continue.", { fontSize: 15, fontWeight: 'normal'}], { width: 400 });
```
Improves readability :-)

Reply:
Agreed, that is nice!